selfdrive/car/chrysler/interface.py

```diff
@@ -70,11 +60,13 @@ def get_params(candidate, fingerprint, vin=""):
     ret.lateralTuning.pid.kf = 0.00006   # full torque for 10 deg at 80mph means 0.00007818594
     ret.steerActuatorDelay = 0.1
     ret.steerRateCost = 0.7
+    tire_stiffness_factor = 0.444 # not optimized yet
```

Comment:
seems like this is an unnecessary addition and change?
Should be 1, but we can remove it per my comment above